sdk/communication/Azure.Communication.NetworkTraversal/tests/CommunicationRelayClient/CommunicationRelayClientLiveTests.cs

```diff
@@ -60,6 +60,40 @@ public async Task GettingTurnCredentialsGeneratesTurnCredentials(AuthMethod auth
         }
 
         [Test]
+        [TestCase(AuthMethod.ConnectionString, TestName = "GettingTurnCredentialsWithConnectionStringRouteType")]
```

Comment:
Let's change the other existing tests which don't request a specific routeType to assert that their response includes a `routeType` (no matter which value).